pkg/table-filter/helper.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package filter
+
+import "sort"
+
+// reversing replace the contents of a slice with the same elements but in reverse order.
+func reversing(items interface{}) {
+	sort.SliceStable(items, func(i, j int) bool {
+		return true
+	})
+}
```

Comment:
wut, is this an O(n log n) reverse method 😭

Reply:
Fixed! Thanks for your review!